### ISSUE TITLE
Make the IP address and mask field windows wider

### ISSUE THREAD

**Issue by ostasevych** [nextcloud/limit_login_to_ip#90]
Please, make the field windows where a user enters the IP address and the mask wider. By now the mask doesn't fit the window at all. 
I suggest to make it similar to the window where the white listed IPs are added, see below.

![image](https://github.com/nextcloud/limit_login_to_ip/assets/5199030/9b86c83a-5afb-4e08-8612-74de45820be6)

**Comment by ENTPRESTIGIOUS** [comment]
Yes I agree

**Comment by Altahrim** [comment]
Hello,

Thanks to the work of @pringelmann on Vue 3, the field is now wider in v4.4.0

**Comment by oleua** [comment]
> Hello,
> 
> Thanks to the work of @pringelmann on Vue 3, the field is now wider in v4.4.0

Hi. Can you see the webview of the app somewhere? In my NC33.0.6 it has disappeared after the last update in the admin settings.

**Comment by pringelmann** [comment]
@oleua by webview do you mean the `limit_login_to_ip` section on the admin settings page? If it is not appearing at all could you please open the developer console and check if there are any errors 

EDIT: I tested it through the appstore myself. Looks like CSS bundles are missing. Will fix and open a PR

**Comment by oleua** [comment]
Yes , there's  no completely the section in the admin settings, the js folder is not present at all, and the log contains the relevant error - jsloader.

**Comment by pringelmann** [comment]
> Yes , there's no completely the section in the admin settings, the js folder is not present at all, and the log contains the relevant error - jsloader.

A release just went out that should fix this (`v4.4.1`)
